starfish/test/full_pipelines/api/test_dartfish.py

```diff
@@ -126,7 +127,28 @@ def test_dartfish_pipeline_cropped_data():
         zero_norm_stack.xarray[0, 0, 0, 50:60, 60:70]
     )
 
+    pipeline_log = zero_norm_stack.log
+
+    assert pipeline_log[0]['method'] == 'ScaleByPercentile'
+    assert pipeline_log[1]['method'] == 'ZeroByChannelMagnitude'
+
     spot_intensities = dartfish.initial_spot_intensities
+
+    pipeline_log = spot_intensities.attrs['log']
+
+    assert pipeline_log[0]['method'] == 'ScaleByPercentile'
+    assert pipeline_log[1]['method'] == 'ZeroByChannelMagnitude'
+
+    # Test serialization / deserialization of IntensityTable log
+
+    fp = tempfile.NamedTemporaryFile()
+    spot_intensities.save(fp.name)
```

Comment:
doesn't save convert the log into a json-encoded string? can you assert on the value of the log here?

Reply:
I do right below! I reload and assert the value.

Reply:
I believe that if you reload and assert on the value, the value will be correct.  My concern is that if you encode it on save, and continue to manipulate the IntensityTable, the log will now be serialized, and _not_ correct.

Reply:
ahhhh I understand what you mean. I'll test for that